src/demo/README.md

```diff
@@ -0,0 +1,60 @@
+## Prepare
+### Run docker-compose
+    `docker-compose -f docker-compose.yml up -d`
+
+   It'll start:
+   - Postgres db for the VCAuthn service
+   - Keycloak with a pre-configured vc-authn realm
+
+### Start VCAuthn service
+    `dotnet run`
+
+### Seed Presentation configuration
+```
+POST https://localhost:5001/api/vc-configs
```

Comment:
provide curl comment?

Reply:
I do understand that curl has wider use comparing to `.http` files (which is a format for this code block). However, all examples of large json in curl that I know seem quite hard to digest. 
I think it might be better to strive for readability with these examples. What do you think?

Reply:
After you show me the how the .http file works in IDE, happy as it is now.